src/web/components/Lineup.tsx

```diff
@@ -19,6 +19,10 @@ const Row = ({ children }: { children: React.ReactNode }) => (
     </div>
 );
 
+const BackgroundRed = '#cc2b12';
```

Comment:
I think just keep these inline

Reply:
I was wondering if they should move to palette, actually. Having them inline fails the 'color-no-hex' test in TeamCity:

https://teamcity.gutools.co.uk/viewLog.html?tab=buildLog&buildTypeId=dotcom_rendering&buildId=544694&_focus=149
> Please use the src-foundations palette variables instead of hex values  color-no-hex

Reply:
Looking at a few match reports, it actually looks like these are dynamic and need to be passed down with the match report data https://www.theguardian.com/tone/matchreports

Reply:
@shtukas are these colours on the data model?

Reply:
These are the yellow and red card icons and as far as I am aware their colour [is compiled once in frontend](https://github.com/guardian/frontend/blob/9ae6637/tools/__tasks__/compile/images/icons.js#L56).

https://www.theguardian.com/football/2020/jul/04/derby-nottingham-forest-championship-match-report

I could not find any information in the JSON file that could suggest these are in fact dynamic.

Reply:
Ah ok, yup, leave as is then, I don't think they need to be in palette in this case unless @SiAdcock thinks so

Reply:
Yeah these colours look pretty unique to me, I'd say keep them hardcoded in DCR.